packages/docs/docs/communications/organizing-communications/organizing-communications.md

```diff
@@ -0,0 +1,167 @@
+
+Below is an example of organizing a communication using all three of LOINC, SNOMED, and a custom code.
+
+```
+{
+  resourceType: "Communication",
+	//...
+	category: [
+	  // A category coded using LOINC
+		{
+			text: "Alert",
+			coding: [
+				{
+					code: "74018-3"
+					system: "https://fhir.loinc.org",
+				}
+			]
+		},
+	],
+	//...
+	topic: {
+	  // A topic coded using SNOMED
+		text: "High blood pressure",
+		coding: [
+			{
+				code: "38341003",
+				system: "http://snomed.info/sct",
+				display: "Hypertensive Disorder"
+			}
+		]
+	}
+}
+
+{
+	resourceType: "Communication",
+	//...
+	// An example of how an internal custom coding could be used for both category and topic
+	category: [
+		{
+			text: "Appointment Reminder",
+			coding: [
+				{
+					code: "apt-rem",
+					system: "https://example-practice.org"
+				}
+			]
+		}
+	],
+	//...
+	topic: {
+		text: "Annual Physical"
+		coding: [
+			{
+				code: "annu-phys",
+				system: "https://example-practice.org"
+			}
+		]
+	}
+}
+```
+
+## Search and GraphQL
+
+Effectively organizing your communications makes it easier to search and query for specific data. For example, if you want to get all of the communications between the provider and a specific patient, you could use the following query. In this, we query for a patient by their id, then search all of the communications that reference them. From these communications, we get the text of the communication, the sender and time it was sent, and the receiver and time it was received.
+
+```
+{
+	# Search for a specific patient by id
+  Patient(id: "example-patient-id") {
+    resourceType
+    id
+		name {
+			family
+			given
+		}
+		# Search for communications that reference the patient
+    communications: CommunicationList(_reference: patient) {
+      text {
+        div
+      }
+      sent
+      sender {
+        resource {
+          ... Practitioner
+        }
+      }
+      received
+      recipient {
+        resource {
+          ... Practitioner
+        }
+      }
+    }
+  }
+}
+
+fragment PractitionerDetails on Practitioner {
+	name {
+		family
+		given
+	}
+}
+
+fragment PatientDetails on Patient {
+	name {
+		family
+		given
+	}
+}
+```
+
+Here is an example of how you could search for all communications within your provider that are not linked to an encounter. Using CommunicationList we are able to return a list of all communications that fit our argument of encounter: null. In this case we return the text of the communication, the sender, and the receiver.
+
+```
+{
+	CommunicationList(encounter: null) {
+		text {
+			div
+		},
+		sender {
+			resource {
+				... Practitioner
+			}
+		},
+		recipient {
+			resource {
+				... Patient
+			}
+		}
+	}
+}
+
+fragment PractitionerDetails on Practitioner {
+	name {
+		family,
+		given
+	}
+}
+
+fragment PatientDetails on Patient {
+	name {
+		family,
+		given
+	}
+}
+```
```

Comment:
add permalink to graphiql tool

Reply:
For each example you might include a link like so.  Preview this query in [GraphiQL](https://graphiql.medplum.com/?query=XXXX)